backend/app/views/spree/admin/payments/_list.html.erb

```diff
@@ -57,12 +57,14 @@
             <% end %>
             <% allowed_actions = payment.actions.select { |a| can?(a.to_sym, payment) } %>
             <% allowed_actions.each do |action| %>
-              <% if action == 'credit' %>
+              <% if action == 'credit' && !payment.completed? %>
```

Comment:
@ccarruitero what do you think?

Reply:
I think that the condition should be a bit different Now it's verified when the action is `credit` and the payment is not completed. I think it should be:

```<% if action == 'credit' && payment.completed? %>```

as we want to show the button only when the payment is `complete`